### ISSUE TITLE
kicking from conference calls

### ISSUE THREAD

**Issue by wetfishQA** [wetfish/fishchat#11]
admins should be able to kick people, like people with black cams or shitty mics

**Comment by kelsh** [comment]
I disagree.  Everyone should have muting and hiding.  No one should have kicking.
